lib/school_house_web/templates/post/index.html.eex

```diff
@@ -2,7 +2,7 @@
   <div class="px-4 mx-auto max-w-7xl sm:px-6 lg:px-8">
     <div class="lg:text-center">
       <p class="mt-2 text-4xl font-extrabold tracking-tight leading-8 text-purple dark:text-purple-dark sm:text-4xl">
-      Recent Posts
+        <%= @page_title %>
```

Comment:
```suggestion
        <%= assigns[:blog_heading] || "Recent Posts" %>
```

Reply:
Ah good catch - sorry for not realising before!